services/login_service/app/.gitignore

```diff
@@ -2,4 +2,6 @@ __pycache__/
 auth/__pycache__/
 config/__pycache__/
 email_verification/__pycache__/
-verify_table/__pycache__/
\ No newline at end of file
+verify_table/__pycache__/
```

Comment:
this kind of adding every single __pycache__ directory is not needed 
`*.pyc` should be suficient

Reply:
Also, why is gitignore in there. Why don't you use te top project one?